### PR TITLE
Update Translation of Home Page

### PR DESCRIPTION
**Pull request by zhiyingfang2022**
* **Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
hi～When viewing the document, I found that some content on the homepage has not been translated, so submit this part for translation.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:We can further optimize and adjust the layout and format specifications of the homepage content.

### CODE REVIEW

**Review by kevin-wangzefeng — COMMENTED** [review]
We don't add whitespace between English and Chinese words, there are multiple places with similar preferences. So I'd suggest to keep them as they are.

File: i18n/zh/code.json [hunk 1]
```diff
@@ -328,7 +339,7 @@
     "message": "欢迎来到KubeEdge"
   },
   "A Kubernetes Native Edge Computing Framework": {
-    "message": "Kubernetes原生边缘计算框架"
+    "message": "Kubernetes 原生边缘计算框架"
```
Comment (kevin-wangzefeng) [inline i18n/zh/code.json:342]:
```suggestion
    "message": "Kubernetes原生边缘计算框架"
```

File: i18n/zh/code.json [hunk 2]
```diff
@@ -349,10 +360,10 @@
     "message": "开发人员可以编写常规的基于http或mqtt的应用程序；容纳并在Edge或Cloud的任何地方运行。"
   },
   "Kubernetes-Native": {
-    "message": "Kubernetes原生"
+    "message": "Kubernetes 原生"
```
Comment (kevin-wangzefeng) [inline i18n/zh/code.json:363]:
```suggestion
    "message": "Kubernetes原生"
```

File: i18n/zh/code.json [hunk 3]
```diff
@@ -349,10 +360,10 @@
...
   },
   "KubeEdge is built upon Kubernetes. Users can orchestrate apps, manage devices and monitor app and device status on Edge nodes just like a traditional Kubernetes cluster in the Cloud.": {
-    "message": "KubeEdge基于kubernetes构建，为云和边缘之间的网络，应用程序管理和元数据同步提供基础架构支持。"
+    "message": "KubeEdge 基于kubernetes 构建，为云和边缘之间的网络，应用程序管理和元数据同步提供基础架构支持。"
```
Comment (kevin-wangzefeng) [inline i18n/zh/code.json:366]:
```suggestion
    "message": "KubeEdge基于kubernetes构建，为云和边缘之间的网络，应用程序管理和元数据同步提供基础架构支持。"
```

**Review by kevin-wangzefeng — COMMENTED** [review]
minor comments

**Comment by zhiyingfang2022** [comment]
> minor comments
Sorry, I didn't know the rules very well before. I submitted multiple commits, which have now been merged. At the same time, the problem of Chinese and English spaces in the full text has been modified.

**Comment by zhiyingfang2022** [comment]
This PR is out of date and therefore closed.

**Pull request closed without merging**